Objective-C/CBLReplicator.h

```diff
@@ -153,6 +153,25 @@ typedef struct {
  */
 - (void) removeChangeListenerWithToken: (id<CBLListenerToken>)token;
 
+/**
+ Gets a set of document Ids, who have revisions pending push. This API is a snapshot and results may
+ change between the time the call was made and the time the call returns.
+ 
+ @param error error On return, the error if any.
+ @return A  set of document Ids, each of which has one or more pending revisions. If error, nil.
+ */
+- (nullable NSSet<NSString*>*) pendingDocumentIds: (NSError**)error;
```

Comment:
I think it's the method is pendingDocumentIDs per the spec.

Reply:
That would be contrary to Java naming conventions.  Do we really all have to standardize on the conventions of some language?  Not being sarcastic, here.  I don't understand the advantage of imposing some other language's conventions.  I am probably missing something...

Reply:
In general, each platform can adjust the name to what makes sense to its naming conversion. For example, Objective-C has documentWithID: and Java has getDocument(String id). Please feel free to make this kind of adjustment. It might be a good idea to double check if Java has use xxxID() previously (I couldn't think anything) so we could also weight on that.

@borrrden @Sandychuang8 what do you think about this? What do you normally do on .NET especially the ID thing?

Reply:
Good point.  Java *should* use `pendingDocumentIds`, but I'll check.

Reply:
If it's a property, .Net will use PendingDocumentIds, GetPendingDocumentIds if it's a method.